server/src/Administration/Domain/Supplier/Handler/EditSupplierHandler.php

```diff
@@ -42,7 +42,10 @@ public function __invoke(EditSupplier $command): void
 
     private function updateSupplier(EditSupplier $command, Supplier $supplier): Supplier
     {
-        if ($supplier->name() !== $command->name()) {
+        if ($supplier->name() !== $command->name()->getValue()) {
```

Comment:
Why sometimes name is a scalar and sometimes it's a VO? I think it's better to use VO as much as possible. But maybe I miss something. The same for the others.

Reply:
I review others to make all with VO :+1:

Reply:
After visu, I wonder if you are talking about the fact that 
 - the Command calls an VO, while the model calls a scalar; 
 - or do you want the model to have only VOs and the 'getters' return the scalars? 
:thinking: